src/pages/Profile.js

```diff
@@ -1,11 +1,151 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useContext } from "react";
+import { SkynetContext } from "../state/SkynetContext";
 import Container from "@material-ui/core/Container";
+import BlogPreviewProfile from "../components/BlogPreviewProfile";
+import { useParams } from "react-router-dom";
+import { loadBlogProfile } from "../data/feedLibrary";
+import Typography from "@material-ui/core/Typography";
+import Skeleton from "@material-ui/lab/Skeleton";
+import { displayName } from "../data/displayName";
+import Divider from "@material-ui/core/Divider";
+import Grid from "@material-ui/core/Grid";
+import Avatar from "@material-ui/core/Avatar";
+import IconButton from "@material-ui/core/IconButton";
+import Twitter from "@material-ui/icons/Twitter";
+import GitHub from "@material-ui/icons/GitHub";
+import Telegram from "@material-ui/icons/Telegram";
+import Reddit from "@material-ui/icons/Reddit";
+import Facebook from "@material-ui/icons/Facebook";
 
 //Profile page component, used to view a users blogs in a feed
 export default function Profile(props) {
+  const { id } = useParams(); //get params from route, id = 'ed25519-' + user's MySky userID
```

Comment:
Same comment as on previous PRs. It would be nicer to have the comments in a single comment block above this block of code instead of on the end of the lines.